scripts/js_test.sh

```diff
@@ -15,7 +15,7 @@ else
 fi
 
 export FILE_PATTERN=${1:-'"src/**/*_test.js"'}
-CMD_ARGS="--require @babel/register $FILE_PATTERN"
+CMD_ARGS="$FILE_PATTERN"
```

Comment:
are we going to bother with coverage and so on in this project? if no we can delete this whole file and call mocha directly from the `package.json`

Reply:
I think we should have coverage and @gumaerc is working on it now. It would be useful because I think the complicated business logic will need significant testing.